### ISSUE TITLE
see what can be blocked on the following sites

### ISSUE THREAD

**Issue by abidjauhar17** [hagezi/dns-blocklists#2055]
Can I suggest that you check this e-commerce site from Indonesia to see if there is tracking or something like that that can be blocked?

https://www.tokopedia.com/
![image](https://github.com/hagezi/dns-blocklists/assets/37255346/c6afe633-af3a-44cd-b71d-2a53c44f348f)

**Comment by hagezi** [comment]
Thanks, I'll take a look. 

What is currently blocked: 
```
www.google-analytics.com
api2.branch.io
js-agent.newrelic.com
sslwidget.criteo.com
www.googletagmanager.com
cdn.branch.io
adservice.google.com
googleads.g.doubleclick.net
certify-js.alexametrics.com
```

Log:
```
01/19/2024 4:50:20 PM | Blocked | www.google-analytics.com    | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:20 PM | Blocked | api2.branch.io              | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:20 PM | Blocked | js-agent.newrelic.com       | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:20 PM | Blocked | js-agent.newrelic.com       | A     | x-hagezi-ultimate
01/19/2024 4:50:20 PM | Blocked | api2.branch.io              | A     | x-hagezi-ultimate
01/19/2024 4:50:20 PM | Blocked | api2.branch.io              | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:20 PM | Blocked | www.google-analytics.com    | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:20 PM | Blocked | www.google-analytics.com    | A     | x-hagezi-ultimate
01/19/2024 4:50:20 PM | Blocked | js-agent.newrelic.com       | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | sslwidget.criteo.com        | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | www.googletagmanager.com    | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | sslwidget.criteo.com        | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | cdn.branch.io               | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | cdn.branch.io               | A     | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | cdn.branch.io               | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | adservice.google.com        | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | adservice.google.com        | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | adservice.google.com        | A     | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | sslwidget.criteo.com        | A     | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | www.googletagmanager.com    | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | googleads.g.doubleclick.net | A     | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | googleads.g.doubleclick.net | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | googleads.g.doubleclick.net | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | certify-js.alexametrics.com | AAAA  | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | certify-js.alexametrics.com | A     | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | certify-js.alexametrics.com | HTTPS | x-hagezi-ultimate
01/19/2024 4:50:19 PM | Blocked | www.googletagmanager.com    | A     | x-hagezi-ultimate
```

**Comment by abidjauhar17** [comment]
> Thanks @abidjauhar17, I'll take a look.
> 
> What is currently blocked:
> 
> ```
> www.google-analytics.com
> api2.branch.io
> js-agent.newrelic.com
> sslwidget.criteo.com
> www.googletagmanager.com
> cdn.branch.io
> adservice.google.com
> googleads.g.doubleclick.net
> certify-js.alexametrics.com
> ```
> 
> Log:
> 
> ```
> 01/19/2024 4:50:20 PM | Blocked | www.google-analytics.com    | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:20 PM | Blocked | api2.branch.io              | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:20 PM | Blocked | js-agent.newrelic.com       | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:20 PM | Blocked | js-agent.newrelic.com       | A     | x-hagezi-ultimate
> 01/19/2024 4:50:20 PM | Blocked | api2.branch.io              | A     | x-hagezi-ultimate
> 01/19/2024 4:50:20 PM | Blocked | api2.branch.io              | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:20 PM | Blocked | www.google-analytics.com    | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:20 PM | Blocked | www.google-analytics.com    | A     | x-hagezi-ultimate
> 01/19/2024 4:50:20 PM | Blocked | js-agent.newrelic.com       | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | sslwidget.criteo.com        | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | www.googletagmanager.com    | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | sslwidget.criteo.com        | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | cdn.branch.io               | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | cdn.branch.io               | A     | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | cdn.branch.io               | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | adservice.google.com        | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | adservice.google.com        | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | adservice.google.com        | A     | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | sslwidget.criteo.com        | A     | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | www.googletagmanager.com    | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | googleads.g.doubleclick.net | A     | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | googleads.g.doubleclick.net | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | googleads.g.doubleclick.net | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | certify-js.alexametrics.com | AAAA  | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | certify-js.alexametrics.com | A     | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | certify-js.alexametrics.com | HTTPS | x-hagezi-ultimate
> 01/19/2024 4:50:19 PM | Blocked | www.googletagmanager.com    | A     | x-hagezi-ultimate
> ```

Is it also blocked in the PRO filter?

**Comment by hagezi** [comment]
@abidjauhar17 Yes, except `googletagmanager.com`, this domain is only blocked in the aggressive lists (Pro++ and Ultimate) because it leads to functional restrictions on various pages.

During the analysis, I found another potential tracker that has not yet been blocked: `ta.tokopedia.com`

Apart from that, I have not found anything else that can be blocked. The following domains are still in the lists, but they were not called up during my test:
```
analytics.tokopedia.com
aypbpr.tokopedia.com
log.tokopedia.net
metric.tokopedia.com
```

**Comment by abidjauhar17** [comment]
> @abidjauhar17 Yes, except `googletagmanager.com`, this domain is only blocked in the aggressive lists (Pro++ and Ultimate) because it leads to functional restrictions on various pages.
> 
> During the analysis, I found another potential tracker that has not yet been blocked: `ta.tokopedia.com`
> 
> Apart from that, I have not found anything else that can be blocked. The following domains are still in the lists, but they were not called up during my test:
> 
> ```
> analytics.tokopedia.com
> aypbpr.tokopedia.com
> log.tokopedia.net
> metric.tokopedia.com
> ```

Can you block trackers that are not needed but don't cause problems with the website?

**Comment by hagezi** [comment]
I have checked the payload from `ta.tokopedia.com`. It is a tracker that tracks what you have clicked on and from where the click was made. I have not noticed any restrictions so far. I will block this tracker in the next update.

The rest is already blocked in the lists.

**Comment by hagezi** [comment]
added.

**Comment by abidjauhar17** [comment]
> added.

Okay. May I know with what tool you can detect trackers on a website?

and can I suggest checking on other websites? If possible, should I create a new issue or just go straight to this issue

**Comment by hagezi** [comment]
This is all manual work, I looked at the network traffic with the Firefox developer tools and looked at the payload of each domain. With a certain amount of intuition you can recognize potential trackers.

I can of course check more sites, but most of them will be covered and it's quite time-consuming and not always as quick as in this case.

**Comment by abidjauhar17** [comment]
> This is all manual work, I looked at the network traffic with the Firefox developer tools and looked at the payload of each domain. With a certain amount of intuition you can recognize potential trackers.
> 
> I can of course check more sites, but most of them will be covered and it's quite time-consuming and not always as quick as in this case.

it doesn't matter if it's fast or slow. as long as the tracker and analytics are removed
I appreciate your efforts

**Comment by abidjauhar17** [comment]
@hagezi try to see what can be blocked here
https://www.vidio.com/

example of one video as a reference
https://www.vidio.com/watch/7393292-episode-01

![image](https://github.com/hagezi/dns-blocklists/assets/37255346/849e7b38-b723-40f0-8d0a-dee00295b954)

**Comment by abidjauhar17** [comment]
> @hagezi try to see what can be blocked here https://www.vidio.com/
> 
> example of one video as a reference https://www.vidio.com/watch/7393292-episode-01
> 
> ![image](https://private-user-images.githubusercontent.com/37255346/298378245-849e7b38-b723-40f0-8d0a-dee00295b954.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5ODIzMjYsIm5iZiI6MTcwNTk4MjAyNiwicGF0aCI6Ii8zNzI1NTM0Ni8yOTgzNzgyNDUtODQ5ZTdiMzgtYjcyMy00MGYwLThkMGEtZGVlMDAyOTViOTU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIzVDAzNTM0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIzYjE4ZWQ4NzhjMDY0MWZlNzRhNWZmYTIyNzM2ZTZhZGRiNTk1NjM4M2YwOTNhZDczN2E3MjkxZmVkYzRiYzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RJPesBAEssxwMv6pH4zb-I0CuoY0DhCXnAcB1z9_jIA)

@hagezi

**Comment by hagezi** [comment]
I have not found anything blockable for DNS. I'm not going to investigate any more sites now, I don't have the time at the moment.
